lang/javascript_test.go

```diff
@@ -0,0 +1,112 @@
+package lang
+
+import (
+	"context"
+	"testing"
+
+	"github.com/safedep/code/core"
+	"github.com/safedep/code/core/ast"
+	"github.com/safedep/code/fs"
+	"github.com/safedep/code/parser"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestJavascriptLanguageMeta(t *testing.T) {
+	t.Run("Name", func(t *testing.T) {
+		l := &javascriptLanguage{}
+		assert.Equal(t, javascriptLanguageName, l.Name())
+	})
+
+	t.Run("Code", func(t *testing.T) {
+		l := &javascriptLanguage{}
+		assert.Equal(t, core.LanguageCodeJavascript, l.Meta().Code)
+	})
+
+	t.Run("ObjectOriented", func(t *testing.T) {
+		l := &javascriptLanguage{}
+		assert.True(t, l.Meta().ObjectOriented)
+	})
+}
+
+type ImportExpectations struct {
+	filePath string
+	imports  []ast.ImportJsonString
+}
+
+var importExpectations = []ImportExpectations{
+	{
+		filePath: "fixtures/imports.js",
+		imports: []ast.ImportJsonString{
+			"{ModuleName: express, ModuleItem: , ModuleAlias: express, WildcardImport: false}",
```

Comment:
We can probably create expected nodes here, such as `core.ImportNode{ModuleName: "..", ...}` and using `assert. EqualExportedValues` to compare the two structs

Reply:
`ImportNode` consists of sitter nodes, i don't think we can hard code these in a simple manner

Reply:
Ah ok! Then lets just call it `String()` because thats a common convention in Go for generating a string representation of a struct for introspective. You can maintain the same `{..}` format for the generated string but calling the function`.String()` is more idiomatic Go.